types/custom-functions-runtime/index.d.ts

```diff
@@ -126,13 +126,14 @@ declare namespace CustomFunctions {
         /**
          *
          * This error code indicates that the function used is dividing by zero or empty cells.
+         * Be aware that JavaScript allows division by zero, so you need to write an error handler carefully to detect this condition.
```

Comment:
@mscharlock The wording of this sentence is a bit misleading. By "allows", I think you mean that JavaScript doesn't throw an exception on division by zero. I believe it returns Infinity or NegativeInfinity. I understand the sentiment that is desired, but I wonder if there might be a more precise way to explain it. Just thought I would mention in case you feel that this could be improved.

Reply:
Hi @akrantz, thanks for your comment! My goal with this update was to be consistent with the phrase used [in the docs](https://docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-errors#the-customfunctionserror-object) to describe this error code. I'll consult another writer and see if we can brainstorm a more precise sentence, and if so I'll update it in both the d.t.s and the docs.